floto/templates/base.html

```diff
@@ -15,7 +15,7 @@
       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@5.x/css/materialdesignicons.min.css">
       <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons" rel="stylesheet" type="text/css">
       <link href="https://cdn.jsdelivr.net/npm/quasar@2.12.6/dist/quasar.prod.css" rel="stylesheet" type="text/css">
-
+      <link href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" rel="stylesheet" type="text/css">
```

Comment:
Can you add this to a block called `styles` at the top of the overview template? That way it'll only get loaded on the overview page.